tstrait/genetic_value.py

```diff
@@ -6,6 +9,21 @@
 from .base import _check_instance, _check_dataframe, _check_non_decreasing  # noreorder
 
 
+@dataclass
+class GeneticResult:
+    """Data class that contains effect size dataframe and phenotype dataframe.
+
+    :param effect_size: Effect size dataframe that includes site_id, causal_state,
+        effect_size, and trait_id
+    :param phenotype: Phenotype dataframe that includes trait_id, individual_id,
```

Comment:
These param documentation is incorrect.

Also probably better to document the members individually like

```python
effect_size: pd.DataFrame
"""
Effect size dataframe ...
"""
```

Reply:
I will modify the docstrings by tomorrow.